Follow pwm_apply_might_sleep() rename in gpio-pwm and ir-rx51

peat-psuwit opened the pull request:
[raspberrypi/linux#6018](https://github.com/raspberrypi/linux/pull/6018) introduces commit https://github.com/raspberrypi/linux/commit/b23e6d7dbd3198b38ab131f51ffdfd8986a8c041 ("pwm: Rename pwm_apply_state() to pwm_apply_might_sleep()"), but didn't rename these 2 modules which use the renamed function. Rename them to allow building them in NixOS which tries to build every module as a module.

peat-psuwit committed:
drivers/gpio/gpio-pwm: follow pwm_apply_might_sleep() rename

Fixes: b23e6d7dbd31 ("pwm: Rename pwm_apply_state() to pwm_apply_might_sleep()")
Signed-off-by: Ratchanan Srirattanamet <peathot@hotmail.com>

pelwell commented:
I'll take the first of these commits, but a commit equivalent to your ir-rx51 patch is already present - see [a10c3d5ff9a343c0b898215253750fedc38ac765.](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.34&id=a10c3d5ff9a343c0b898215253750fedc38ac765)

peat-psuwit commented:
I see. The commit with the right content gets included as part of 6.6.33 merge. Sorry for not checking before creating the PR (the branch has been sitting for a while).

I've rebased the branch and removed the second commit.

pelwell commented:
I fixed up the commit message - gpio-pwm is a downstream driver, which is why it wasn't included in the mass rename.
